spec/validators/url_validator_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+class TestModel
+  include ActiveModel::Model
+
+  attr_accessor :test_url
+
+  validates :test_url, url: true
+end
```

Comment:
We created a fake test model for this. Not sure if this is how you want this done @toyhammered 

I figured it was preferable to crafting these examples in every model that uses this validator. But if you had different ideas, definitely let us know.

Reply:
This is perfectly fine. I've seen this pattern used quite a bit and have nothing against it (otherwise it is hard to test validators unless you have some shared_examples that get thrown into every model which is kinda... eehhhhh)